app/models/work.rb

```diff
@@ -846,11 +846,6 @@ def set_word_count(preview = false)
     end
   end
 
-  after_update :remove_outdated_downloads
```

Comment:
Does this mean that more files will be left around ?

Reply:
No, because we delete them after we generate and send them off.